### PR TITLE
Bug fix package_name

### PR DESCRIPTION
**Pull request by NoaAzoulay**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
fix error on packages without "_" in its name 

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by itai1357 — APPROVED** [review]

File: checkov/common/output/cyclonedx.py [hunk 1]
```diff
@@ -196,7 +195,7 @@ def create_library_component(self, resource: Record | ExtraResource, check_type:
         package_name = resource.vulnerability_details["package_name"]
         package_version = resource.vulnerability_details["package_version"]
 
-        if purl_type == PURL_TYPE_MAVEN:
+        if purl_type == PURL_TYPE_MAVEN and '_' in package_name:
```
Comment (nimrodkor) [inline checkov/common/output/cyclonedx.py:198]: @NoaAzoulay Can there be maven packages without `_`? Is it because we are using `:`?

Comment (gruebel) [reply checkov/common/output/cyclonedx.py:198]: for some reason twistcli didn't find the group name for a specific Maven package, even it is has one in Maven. It was this one `bcpkix-jdk15on` which should have been `org.bouncycastle_bcpkix-jdk15on`

Comment (gruebel) [reply checkov/common/output/cyclonedx.py:198]: @NoaAzoulay would be great, if you can add or extend a test to validate the new change 🙂

Comment (NoaAzoulay) [reply checkov/common/output/cyclonedx.py:198]: @gruebel sure, working on it

**Review by gruebel — APPROVED** [review]
🍰

**Pull request merged**